### PR TITLE
Adds image-upload ansible playbook 

### PR DESCRIPTION
**Pull request by Stringy**
## Description

Adds and image-upload ansible playbook to synchronize development Collector image builds with the ansible inventory. Intended primarily as a development tool for those who do not wish to use an intermediary quay repository.

### CODE REVIEW

File: ansible/image-upload.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  vars_files:
+    - group_vars/all.yml
+  hosts: "platform_*:&job_id_{{ job_id }}"
+  strategy: free
+
+  tasks:
+    - name: Get Collector Image
+      shell: make -C "{{ collector_root }}" tag
+      register: collector_tag_result
+      delegate_to: localhost
```
Comment (Molter73) [inline ansible/image-upload.yml:13]: Could we instead control the image tag being used with an env variable? Sometimes you want to use an existing image you built a couple days back, but might not have the branch in the same state anymore.

**Review by Molter73 — COMMENTED** [review]
Awesome! I ended up creating a registry under my quay.io account for this exact situation, having it automated will be nice.

File: ansible/image-upload.yml (hunk 1 above)
Comment (Stringy) [reply ansible/image-upload.yml:13]: yeah that's a good idea. I'll make it default to the current tag if the env var is not provided

File: ansible/image-upload.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+    - set_fact:
+        collector_tag: "{{ collector_tag_result.stdout | replace('\n', '') }}"
+
+    - set_fact:
+        collector_tag: "{{ lookup('env', 'COLLECTOR_TAG') | default(collector_tag) }}"
```
Comment (Molter73) [inline ansible/image-upload.yml:19]: Looks kinda weird that we are setting `collector_tag` twice, maybe we could save the filtered one to a different fact?

**Review by Molter73 — APPROVED** [review]
Just a minor comment on the same fact being set twice, but shouldn't be a blocker for this, so LGTM!

**Pull request merged**